influxdb_iox/src/commands/run/all_in_one.rs

```diff
@@ -213,6 +228,7 @@ pub struct Config {
     )]
     pub persist_max_parallelism: usize,
 
+    // TODO - remove these ingester tuning knobs  from all in one mode??
```

Comment:
I think it would make sense to remove the ingester tuning knobs from the all-in-one configuration but would like to do so as a follow on PR

Reply:
@domodwyer or @carols10cents  do you see any issue with removing what appear to be ingester specific tuning these knobs from all in one mode (this does not affect the knobs on the actual ingester service)

These options appear to have been added in #6811 by @stuartcarnie  when all in one mode was updated to the new kafkaless write path

Reply:
Why do we need to remove them? I can imagine at least the max queue depth & hot partition cost being problematic / needed when running locally.

Reply:
We do not have to remove thee knobs,  the reason I was proposing to do so was to encourage people to use all in one mode for development / testing only. 

I think the idea was that the multi-service (`compactor2`, `ingester2`, etc with a postgres catalog) setup should be used performance sensitive installations where turning was more likely to be needed.

I do not feel strongly about this and if there is value in keeping these knobs exposed then I think we can do it.

Reply:
Ah my comments are not about setting up a performant system or actual deployments - more that the low queue depth might mean you can't write to the system at all depending on what you're writing locally.

Reply:
Got it -- since I don't \ understand what the knobs do in enough detail to understand the implications of replacing them with some constant size, I think it makes sense to leave them as knobs for now. 

```suggestion
    // TODO - evaluate if these ingester knobs could be replaces for some hard coded values
    // as a way to encourage people to use the multi-service setup for performance sensitive
    // installations
```

Reply:
Got it -- since I don't \ understand what the knobs do in enough detail to understand the implications of replacing them with some constant size, I think it makes sense to leave them as knobs for now. 

```suggestion
    // TODO - evaluate if these ingester knobs could be replaces for some hard coded values
    // as a way to encourage people to use the multi-service setup for performance sensitive
    // installations
```

Reply:
Updated comment in https://github.com/influxdata/influxdb_iox/pull/7044